### ISSUE TITLE
FailFast on OS X 10.14.4

### ISSUE THREAD

**Issue by d4ncer** [microsoft/python-language-server#1079]
Hello!

I've been trying to set up `mspyls` on OS X to be used with Emacs via https://github.com/emacs-lsp/lsp-python-ms, but I seem to be running into an issue with the LS crashing on startup.

```
FailFast:
Infinite recursion during resource lookup within System.Private.CoreLib.  This may be a bug in System.Private.CoreLib, or potentially in certain extensibility points such as assembly resolve events or CultureInfo names.  Resource name: Argument_InvalidResourceCultureName

   at System.Environment.FailFast(System.String)
   at System.SR.InternalGetResourceString(System.String)
   at System.SR.GetResourceString(System.String, System.String)
   at System.Globalization.CultureInfo.VerifyCultureName(System.String, Boolean)
   at System.Resources.ResourceManager.GetResourceFileName(System.Globalization.CultureInfo)
   at System.Resources.ManifestBasedResourceGroveler.GrovelForResourceSet(System.Globalization.CultureInfo, System.Collections.Generic.Dictionary`2<System.String,System.Resources.ResourceSet>, Boolean, Boolean, System.Threading.StackCrawlMark ByRef)
   at System.Resources.ResourceManager.InternalGetResourceSet(System.Globalization.CultureInfo, Boolean, Boolean, System.Threading.StackCrawlMark ByRef)
   at System.Resources.ResourceManager.InternalGetResourceSet(System.Globalization.CultureInfo, Boolean, Boolean)
   at System.Resources.ResourceManager.GetString(System.String, System.Globalization.CultureInfo)
   at System.SR.InternalGetResourceString(System.String)
   at System.SR.GetResourceString(System.String, System.String)
   at System.Globalization.CultureInfo.VerifyCultureName(System.String, Boolean)
   at System.Resources.ResourceManager.GetResourceFileName(System.Globalization.CultureInfo)
   at System.Resources.ManifestBasedResourceGroveler.GrovelForResourceSet(System.Globalization.CultureInfo, System.Collections.Generic.Dictionary`2<System.String,System.Resources.ResourceSet>, Boolean, Boolean, System.Threading.StackCrawlMark ByRef)
   at System.Resources.ResourceManager.InternalGetResourceSet(System.Globalization.CultureInfo, Boolean, Boolean, System.Threading.StackCrawlMark ByRef)
   at System.Resources.ResourceManager.InternalGetResourceSet(System.Globalization.CultureInfo, Boolean, Boolean)
   at System.Resources.ResourceManager.GetString(System.String, System.Globalization.CultureInfo)
   at StreamJsonRpc.JsonRpc.ThrowIfConfigurationLocked()
   at StreamJsonRpc.JsonRpc.AddLocalRpcTarget(System.Object, StreamJsonRpc.JsonRpcTargetOptions)
   at Microsoft.Python.LanguageServer.Server.Program.Main(System.String[])
```

I'm not familiar at all with C#, so I'm not even sure where to start debugging this issue. Any help would be appreciated!

**Comment by MikhailArkhipov** [comment]
libicu might be missing. See https://github.com/microsoft/python-language-server/issues/170 for details and possible workaround.

**Comment by jakebailey** [comment]
@d4ncer Can you show the output of these commands?

```
echo $LANG
echo $LC_ALL
```

This seems like dotnet/corefx#30593, which I think is fixed by dotnet/corefx#22378, which is only in the dotnet 3.0 prerelease. If you're compiling this yourself, you might be able to try testing with that new version.

**Comment by d4ncer** [comment]
That was it @jakebailey! I set both to `en_US.UTF-8`, and it seems to be working now. Thanks!
